src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageComparison.java

```diff
@@ -120,4 +123,13 @@ public boolean areImagesTheSame(BufferedImage image1, BufferedImage image2) {
     }
     return true;
   }
+
+  private boolean colorsAreSimilar(Color c1, Color c2){
```

Comment:
Consider to rename method to `areColorsSimilar ` for keeping a convention of this file.